### ISSUE TITLE
The performance of the segmentation network

### ISSUE THREAD

**Issue by johnnylu305** [GrumpyZhou/pytorch-dgcnn#2]
Hi, I ran your code on my machine and got Accuracy: 0.9134 IoU: 0.9859 time 260.4755s after 30 epoch on test set. I might miss something during building the environment for your code so I will rerun it. Anyway, can you tell me the expected accuracy and IoU about your segmentation network?

**Comment by GrumpyZhou** [comment]
Hi, 
Your numbers look good no? What was the issue? So i checked some output i generated that one year ago. One of them is showing me:
````
Namespace(K=20, batch=16, cat='All',
ckpt='output/Shapenet/All/ckpt/ckpt_15.pth', data_dir='data/',
dataset='ShapeNet', epoch=250, gpu=2,
lr=0.001, lrd_factor=0.5, lrd_step=20, network='DGCNNSeg',
odir='model_outputs', seed=7, training=False, val=5,
visenv=None, vishost='localhost', visport=9333, viswin=None,
weight_decay=0.0, worker=2)

Testing DGCNNSeg with ckpt output/Shapenet/All/ckpt/ckpt_15.pth
Accuracy: 0.9038 IoU: 0.9843 time 501.4050s
````
Is this what you want to know?

**Comment by johnnylu305** [comment]
Thank you. Yes, that is what I am looking for. I think your IoU is far higher than the IoU in the paper. You used a different way to calculate IoU, am I right?

**Comment by GrumpyZhou** [comment]
This i am not sure. For the metric, i am using the implementation from [pytorch_geometric mean_iou](https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/utils/metric.html#mean_iou ). If you found any bug in my code, i am happy to know :)
